core/views.py

```diff
@@ -66,6 +69,14 @@ class PostCreateView(LoginRequiredMixin, CreateView):
     template_name = "core/post-create.html"
     login_url = "login"
 
+    def dispatch(self: "PostCreateView", request: HttpRequest, *args: list, **kwargs: dict[str, Any]) -> HttpResponse:
```

Comment:
`*args: list, **kwargs: dict[str, Any]`
means that every object in the tuple `args` must be a `list[???]` and every value in the `kwargs` dict must be a `dict[str, Any]`.
Is that what you meant?

The fact that `*args` is inferred from the variadic token `*`, so applies to `**kwargs`. We only annotate the _values_ there, but not the entire structure (unless in `*args` you're doing some wild `Unpack[tuple[...]]` stuff).

Reply:
In case it's not you meant, you can use `Any` instead of `list`/`dict[str, Any]`.
Judging by https://github.com/typeddjango/django-stubs/blob/f89454da6b2e932577cd9356f4424832980c560d/django-stubs/views/generic/base.pyi#L25, you don't have a better option. On a side note, I think they should have unpacked some typed dicts there.

Reply:
Great suggestion with using `Any`! Thanks